data/shopify_liquid/tags.yml

```diff
@@ -8,6 +8,8 @@
 - cycle
 - decrement
 - echo
+- else
```

Comment:
I'm wondering, should be put all the `endif`, `endcomment` here so we get completion for those too?

And `where` too (from watching your demo).

Reply:
Yeah I have something coming in the fix for #181. Could put that in here too.

Reply:
https://github.com/Shopify/theme-check/commit/369c5abb686c3e5221438aeb9e0d3c274584a9cf

I also make it so you can complete them but only when they make sense.